inference-engine/include/ie_iexecutable_network.hpp

```diff
@@ -173,6 +173,9 @@ class IExecutableNetwork : public details::IRelease {
      * @return code of the operation. InferenceEngine::OK if succeeded
      */
     virtual StatusCode GetContext(RemoteContext::Ptr& pContext, ResponseDesc* resp) const noexcept = 0;
+
+protected:
+    ~IExecutableNetwork() = default;
```

Comment:
Maybe do we need to have virtual destructor here?

Reply:
no